src/OrchardCore.Commerce.Web/OrchardCore.Commerce.Web.csproj

```diff
@@ -7,6 +7,8 @@
     <NodeJsExtensionsMarkdownAnalysisMode>solution</NodeJsExtensionsMarkdownAnalysisMode>
     <ExecDotnetPostcleanCommand>false</ExecDotnetPostcleanCommand>
     <ExecDotnetPrebuildCommand>false</ExecDotnetPrebuildCommand>
+    <NodeJsExtensionsGlobalESLintConfigurationDirectory></NodeJsExtensionsGlobalESLintConfigurationDirectory>
+    <NodeJsExtensionsCreateStylelintConfigurationFile></NodeJsExtensionsCreateStylelintConfigurationFile>
```

Comment:
The config files in the repo are now unneeded.

Reply:
Hmm, nope they are still created.
![image](https://user-images.githubusercontent.com/4007293/183499592-2cc0d2f6-7a32-49ba-b4a5-bcb2c5e927bc.png)

Reply:
@0liver this shouldn't happen, no?

Reply:
Correct - I'd expect them not to be created.

Reply:
Are you sure? I don't think it should work.
Based on [this](
https://github.com/Lombiq/NodeJs-Extensions/blob/fa08be84eac2eeb525a6cb43ed91df81bc60e276/Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.targets#L106-L110) it seems to me that if you set `NodeJsExtensionsGlobalESLintConfigurationDirectory` to empty it means that the path `""` never exists (because it's an invalid path), so the config file is always copied if it doesn't exist.

Reply:
You're correct. 😏

Reply:
Well, then this is as far as we can go here.

Reply:
Please extend the condition there like so:

```xml
    <Copy SourceFiles="$(NodeJsExtensionsNpmPackageSourcePath)/config/consumer/.eslintrc.project.json"
          DestinationFiles="$(MSBuildProjectDirectory)/.eslintrc.json"
          Condition="
            '$(NodeJsExtensionsGlobalESLintConfigurationDirectory)' != ''
            AND !Exists($(NodeJsExtensionsGlobalESLintConfigurationDirectory))
            AND !Exists('$(MSBuildProjectDirectory)/.eslintrc.json')" />
```

Reply:
That was my intention when I wrote that.